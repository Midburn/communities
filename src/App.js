import React, { Component } from 'react';
import { BrowserRouter } from 'react-router-dom';
import './App.scss';
import { Layout } from './components/Layout';
import { withNamespaces } from 'react-i18next';

class App extends Component {

    render() {
        return (
            <div className="App">
                <BrowserRouter>
                        <Layout />
                </BrowserRouter> 
            </div>
        );
    }
}

export default withNamespaces()(App);