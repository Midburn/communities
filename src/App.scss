@import "./styles/_colors.scss";
body {
  height: 100%;
  background: url("assets/images/bg.jpg") no-repeat center center fixed;
  background-size: cover;
  overflow-y: scroll;
  width: 100%;
}

#root, .App, .Layout {
  height: 100%;
}

.clickable {
  cursor: pointer;
}

*::-webkit-scrollbar {
  width: 0.5em;
  height: 0.5em;
}

*::-webkit-scrollbar-track {
  -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
}

*::-webkit-scrollbar-thumb {
  background-color: $yellow;
  outline: 1px solid $yellow;
  border-radius: 100px;
}

.MainPanel {
  transition: 0.8s;
  min-height: 100vh;
  padding-top: 16vh;
  padding-bottom: 16vh;
}


.disabled {
   pointer-events: none;
   opacity: 0.3 !important;
 }