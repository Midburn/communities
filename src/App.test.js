import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import { shallow } from 'enzyme';

describe('App Component', () => {
  it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<App />, div);
        ReactDOM.unmountComponentAtNode(div);
    });
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<App debug />);
        expect(component).toMatchSnapshot();
    });
});