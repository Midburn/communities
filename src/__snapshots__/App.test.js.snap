// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should render correctly in "debug" mode 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
